components/global/portal/io.cellery.observability.ui/src/components/common/DependencyGraph.js

```diff
@@ -298,7 +310,8 @@ DependencyGraph.propTypes = {
     onClickNode: PropTypes.func,
     onClickGraph: PropTypes.func,
     graphType: PropTypes.string,
-    viewGenerator: PropTypes.func
+    viewGenerator: PropTypes.func,
+    loadCellIngress: PropTypes.func
```

Comment:
The name should change. This is to render the label and not specific to ingresses.

Suggestion :- renderNodeLabel